data-processing-lib/ray/pyproject.toml

```diff
@@ -1,23 +1,17 @@
 [project]
-name = "data_prep_toolkit"
-version = "0.1.1"
+name = "data_prep_toolkit_ray"
+version = "0.2.0"
 requires-python = ">=3.10"
-description = "Data Preparation Toolkit Library"
+description = "Data Preparation Toolkit Library for Ray"
 license = {text = "Apache-2.0"}
 readme = {file = "README.md", content-type = "text/markdown"}
 authors = [
     { name = "David Wood", email = "dawood@us.ibm.com" },
     { name = "Boris Lublinsky", email = "blublinsky@ibm.com" },
 ]
 dependencies = [
+    "data-prep-toolkit",
     "ray[default]==2.9.3",
-    "pyarrow==15.0.2",
-    "boto3==1.34.69",
-    "argparse",
-    "mmh3",
-    # These two are to fix security issue
-    "fastapi>=0.109.1",
-    "pillow>=10.2.0",
```

Comment:
Removed these 2 will create security issues in the image

Reply:
I moved these from the python toml back to the ray toml.